pkg/kv/kvserver/replica_test.go

```diff
@@ -15235,15 +15235,15 @@ func TestOverlapsUnreplicatedRangeIDLocalKeys(t *testing.T) {
 		// Point spans not overlapping with unreplicated local RangeID spans.
 		{span: s(roachpb.KeyMin, nil)},
 		{span: s(keys.LocalRangeIDPrefix.AsRawKey().Prevish(1), nil)},
-		{span: s(keys.MakeRangeIDUnreplicatedPrefix(1).Prevish(1), nil)},
 		{span: s(keys.RangeForceFlushKey(1), nil)},
+		{span: s(keys.MakeRangeIDUnreplicatedPrefix(1).Prevish(1), nil)},
+		{span: s(keys.RangeTombstoneKey(1), nil)},
```

Comment:
Would be good to also have `ReplicaID` key because the code special cases it.
And a test for a span that starts at one of the 2 special keys, but also has a non-nil `EndKey`. This should be a `notOk` case.